src/ssl.c

```diff
@@ -81,6 +81,7 @@
     #include <wolfssl/wolfcrypt/idea.h>
     #include <wolfssl/wolfcrypt/curve25519.h>
     #include <wolfssl/wolfcrypt/ed25519.h>
+    #include <wolfssl/openssl/asn1.h>
```

Comment:
openssl/asn1.h should probably go in above block with other openssl compatibility headers.